src/_cffi_src/openssl/cryptography.py

```diff
@@ -5,6 +5,11 @@
 from __future__ import absolute_import, division, print_function
 
 INCLUDES = """
+/* define our OpenSSL API compatibility level to 1.0.0. Any symbols older than
+   that will raise an error during compilation. We can raise this number again
+   after we drop 1.0.2 support in the distant future */
+#define OPENSSL_API_COMPAT 0x10000000L
```

Comment:
Is there a reason we shouodn't set this to 1.0.1, given that's our minimum version?